packages/vuetify/types/index.d.ts

```diff
@@ -254,3 +254,5 @@ export type DataItemsPerPageOption = (number | {
   text: string
   value: number
 });
+
+export type RowClassProp = (item: any) => null|undefined|string|string[]|Record<string, boolean>
```

Comment:
```suggestion
export type RowClassFunction = (item: any) => null|undefined|string|string[]|Record<string, boolean>
```

to be in line with https://github.com/vuetifyjs/vuetify/blob/b8677ad0d3237bc1fe935493c1dd00d1cb73bd2f/packages/vuetify/types/index.d.ts#L236 and others